config.yml

```diff
@@ -1,30 +1,38 @@
----
```

Comment:
Most of the file looks altered because I removed the `---` at the top (it didn't break anything as far as I can tell) and I de-indented everything so that the top-level keys are "flush" with the left margin.

Reply:
Yup, great! Since we don't have any YAML directives in our config file, the presence (or absence)  of the dashes ("---") doesn't make a difference here.